docs/Makefile

```diff
@@ -0,0 +1,192 @@
+# Makefile for Sphinx documentation
```

Comment:
IMO drop the entire docs/ directory. This will be a small package so README is the only documentation we need. This is what we're going to support for junit, tap and junit.xml plugins.